### ISSUE TITLE
[Enhancement]: indicator for entrance rando room checks

### ISSUE THREAD

**Issue by foxsouns** [kbranch/Magpie#15]
Okay, let me elaborate. Let's say, uh, Under the River shows up, and you don't have boots yet: no big deal. Move on, and later on, you find boots: Under the River's exit is now available, however, in-tracker, this is not shown. Maybe we could do a blue door-shaped check for a now available entrance?

**Comment by foxsouns** [comment]
actually, maybe we could have the exclamation mark for "not able to continue", and a question mark for "able, but not checked"

**Comment by kbranch** [comment]
Yeah, that would definitely be good to have. The exclamation mark was a stopgap until something like this was in place. I think the full solution should involve the difficulty indicators since there are glitchy/hard ways to get past a lot of connector obstacles.

**Comment by foxsouns** [comment]
so, there needs to be a distinction between "there is an item in this subarea" and "you can go past this subarea": maybe have a smaller "logic-shape" (the currently used ones) for items in the top right corner, and the entrance itself being a logic-shape inside the "door" frame? i dont want to break the whole "colorblind-safe" thing that the logic-shapes bring to the table.

**Comment by kbranch** [comment]
Yeah, the difficulty shapes are neat, but definitely add some design challenges once you start layering things, lol.

It can be tough to fit a lot of detail into such a small icon, but using position relative to other icons does sound promising - definitely worth experimenting with!
